proxy/pkg/webdav/response.go

```diff
@@ -0,0 +1,77 @@
+package webdav
+
+import (
+	"encoding/xml"
+	"net/http"
+)
+
+type code int
+
+const (
+	// SabredavBadRequest maps to HTTP 400
+	SabredavBadRequest code = iota
```

Comment:
we are duplicating code from reva here, right? cant we reuse that?

Reply:
nope, it's under the internal package...

Reply:
[here](https://github.com/cs3org/reva/blob/f8b91e12b75329371e48b0d11c99c6416f0623b0/internal/http/services/owncloud/ocdav/error.go)

Reply:
We could however abstract the webdav response handling to an external package, but that's out of the scope of this PR IMHO. I can open a ticket as a "good first issue" sort of tasks